includes/multi-currency/class-user-settings.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * WooCommerce Payments Multi Currency User Settings
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Multi_Currency;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that add Multi Currency settings to user my account page.
+ */
+class User_Settings {
+
+	/**
+	 * Multi-Currency instance.
+	 *
+	 * @var Multi_Currency
+	 */
+	protected $multi_currency;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Multi_Currency $multi_currency The Multi_Currency instance.
+	 */
+	public function __construct( Multi_Currency $multi_currency ) {
+		$this->multi_currency = $multi_currency;
+
+		add_action( 'woocommerce_edit_account_form', [ $this, 'add_default_currency_switch' ] );
+		add_action( 'woocommerce_save_account_details', [ $this, 'save_default_currency' ] );
+	}
+
+	/**
+	 * Add a select to allow user choose default currency in `my account > account details`.
+	 *
+	 * @return void
+	 */
+	public function add_default_currency_switch() {
+		?>
+		<p class="woocommerce-form-row woocommerce-form-row--first form-row form-row-first">
+			<label for="wcpay_selected_currency"><?php esc_html_e( 'Default currency', 'woocommerce-payments' ); ?></label>
```

Comment:
We may need to wrap this in the `__()` function call to make it translatable, like we do in [this example](https://github.com/Automattic/woocommerce-payments/blob/d33162013f98e571257e0836a3dda5a4d3aae9fd/woocommerce-payments.php#L121-L128).

Reply:
I thought that [esc_html_e](https://developer.wordpress.org/reference/functions/esc_html_e/) already translate the string. Isn't the equivalent of echo + escape html + translate?

Reply:
You're right! Sorry, I got mixed up with the `esc_html` and `esc_html_e` functions 🤦‍♂️

Reply:
🙂 There are too many similar functions `esc_html`, `esc_html__` and `esc_html_e`. I get used to them faster thanks to VS Code and methods docs.